scripts/server.js

```diff
@@ -13,6 +13,10 @@ const app = createServer({
   rootPath: path.join(__dirname, '..'),
 });
 
+const printDivider = () => {
+  console.log('\n' + '#'.repeat(60) + '\n');
```

Comment:
`70` was used so that it was longer than the text inside.

Reply:
I just reduced the width of all text (in a separate commit). It won't be perfect on every screen but keeping it less wide makes it slightly more versatile.